.github/workflows/release-if-needed.yml

```
@@ -98,19 +101,14 @@ jobs:
         run: |
           python -m pip install --upgrade pip
           pip install hatch
-      - name: Bump version to next patch
+      - name: Push bumped version to main
         run: |
-          formatted_date=$(date +"%Y-%m-%d/%H-%M-%S")
           cd "${{ needs.check-version.outputs.package_name }}"
-          git checkout -b "release/version_bump_next_minor/${formatted_date}"
           hatch version patch,dev
           hatch run changelog:add
           hatch run schema
           git config user.email "145135826+vizro-svc@users.noreply.github.com"
           git config user.name "Vizro Team"
           git add -A
-          git commit -m "version bump"
-          git push --set-upstream origin release/version_bump_next_minor/${formatted_date}
-          ./../tools/version-bump.sh mckinsey vizro ${{ secrets.GITHUB_TOKEN }} \
-          ${{needs.check-version.outputs.package_name}} \
-          ${{needs.check-version.outputs.package_version}} release/version_bump_next_minor/${formatted_date}
+          git commit -m "version bump after ${{needs.check-version.outputs.package_name}}-${{needs.check-version.outputs.package_version}} [skip ci]"
```

```suggestion
          git commit -m "Bump ${{needs.check-version.outputs.package_name}} version to ${{hatch version}} [skip ci]"
```

Not sure what the right syntax here is but basically the output of `hatch version` shell command gives the new version, and it would be nice to have that in the commit message rather than the old version.

---

So please don't accept this suggestion without testing it out and correcting the `${{ hatch version }}` to whatever it should be 😀

---

Done